lib/matplotlib/axis.py

```diff
@@ -774,6 +774,10 @@ def _set_scale(self, value, **kwargs):
         self.isDefault_majfmt = True
         self.isDefault_minfmt = True
 
+    def _get_shared_axes_siblings(self, name):
```

Comment:
I think this only ever gets called with name=self._get_axis_name(), so I'd just get rid of that argument and recompute it in the function body everytime (I doubt this is any significant bottleneck).

Reply:
That was my first approach, but then I realized that `name` was available/required to be available anyway everywhere it was called... (There was for a short period of time a version with `name=None` and in that case looking it up.)

But I can of course change.